test/replication-luatest/gh_7486_prohibit_roll_back_due_to_replication_synchro_timeout.lua

```diff
@@ -0,0 +1,61 @@
+local t = require('luatest')
```

Comment:
This test isn't run by test-run, because it doesn't have a `_test.lua` suffix, like the other one. Please rename it accordingly.